font/bmfont.ksy

```diff
@@ -0,0 +1,155 @@
+meta:
+  id: bmfont
+  title: BMFont binary file
+  file-extension: fnt
+  license: CC0-1.0
+  endian: le
+  bit-endian: le
+doc-ref: https://www.angelcode.com/products/bmfont/doc/file_format.html  
+seq:
+  - id: magic
+    contents: ['BMF', 3]
+  - id: info
+    type: info
+  - id: common
+    type: common 
+  - id: pages
+    type: pages
+  - id: chars
+    type: chars
+  - id: kernings
+    type: kernings
+    if: not _io.eof
+types:
+  info:
+    seq:
+      - id: magic
```

Comment:
Looks like a TLV pattern.

Reply:
This is definitely a TLV pattern, but I couldn't figure out how to make first 4 blocks required and in the correct order, except to explicitly list the blocks.

Reply:
Just insert them and use `valid`